### ISSUE TITLE
Bottom border on tabs says 3px, React implementations recently changed to 2px

### ISSUE THREAD

**Issue by gillibrand** [carbon-design-system/carbon-website#1055]
I assume the docs are behind, but not sure.

Note that 2px looks much better since it's the same width as the focus ring, the the bottom border doesn't appear to shrink and grow when gaining focus anymore.

**Comment by tw15egan** [comment]
Yes, the docs on the website should be updated. Related PR is here:

https://github.com/carbon-design-system/carbon/pull/5482

**Comment by gillibrand** [comment]
Any chance of fixing this earlier than later? It's a small thing, but causing inconsistencies in some teams updating their existing UI-toolkits for a Carbon look.

**Comment by tw15egan** [comment]
PR has been merged, should be included in `10.11.0` set to go out on 4/7

https://github.com/carbon-design-system/carbon/wiki/Release-radar

I can make a PR to update the website now
